src/sdp.ts

```diff
@@ -85,13 +86,12 @@ function certhashToFingerprint(ma: Multiaddr): string {
 
 function ma2sdp(ma: Multiaddr, ufrag: string): string {
   return ANSWER_SDP_FORMAT.replace('%s', ipv(ma))
-    .replace('%s', ip(ma))
-    .replace('%s', ipv(ma))
-    .replace('%s', ip(ma))
-    .replace('%d', port(ma).toString())
-    .replace('%s', ufrag)
-    .replace('%s', ufrag)
-    .replace('%s', certhashToFingerprint(ma));
+    .replaceAll('{IP}', ip(ma))
```

Comment:
You could avoid this replacing by using builtin string interpolation and/or tagged templates: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals